source/simulator/parameters.cc

```diff
@@ -299,6 +299,10 @@ namespace aspect
                        "zero, i.e., we immediately start with the better but more expensive "
                        "preconditioner.");
 
+    prm.declare_entry ("Number of expensive Stokes solver steps", "1000",
+                       Patterns::Integer(0),
+                       "Number of steps used in the expensive Stokes solver.");
```

Comment:
I don't think this adequately conveys the meaning of the parameter. The parameter really means the *maximum* number of expensive solver steps that can be used to reach convergence. Could you reword the description? Would it be worth also updating the *name* of the parameter to something like "Maximum number of expensive..."?